DevExtreme CLI: add React Next.js info (#7295)
DevExpress/devextreme-documentation: concepts/50 React Components/50 Application Template/45 Authentication/00 Authentication.md

> The DevExtreme React Application Template offers authentication UI and functions, including multiple authentication pages. These pages allow users to sign in, create accounts, and reset or change their passwords. The authentication functionality of the Vite and Next.js templates are different.

Reviewer: I think "pages" should be closer to "UI" than to "functions". 

```suggestion
The DevExtreme React Application Template includes authentication functions and UI (multiple authentication pages). These pages allow users to sign in, create accounts, and reset or change their passwords. The authentication functionality of Vite and Next.js templates is different.
```
